action.yml

```
@@ -95,6 +95,9 @@ inputs:
   use_legacy_upload_endpoint:
     description: 'Use the legacy upload endpoint'
     required: false
+  use_oidc:
+    description: 'Use OIDC instead of token. This will ignore any token supplied'
+    required: false
```

```suggestion
    default: false
```

Could you add some documentation in `README.md` on how this is supposed to be used, i.e. does the user have to set anything special in the workflow, e.g. `pypi-publish` requires:
```yaml

    environment:
      name: pypi
      url: https://pypi.org/p/<your-pypi-project-name>
    permissions:
      id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing
```

If the user does not have to set anything, it would be good if there was either a transition announcement, or details on when to use one or the other. From the recent announcement in `v4` it seems that previously, using non-token authentication caused rate-limiting issues. It is unclear if this method will have similar rate limiting issues.

---

@LecrisUT absolutely, this PR is here currently to help ensure OIDC use works on our backend.